proposed/event-dispatcher-meta.md

```diff
@@ -39,48 +39,48 @@ The Working Group identified four possible workflows for event passing, based on
 
 On further review, the Working Goup determined that:
 
-* Collection was a special case of Object enhancement (the collection being the object that is enhanced)
-* Alternative chain is similarly a special case of Object enhancement, as the signature is identical and the dispatch workflow is nearly identical, albeit with an extra check included.
+* Collection was a special case of object enhancement (the collection being the object that is enhanced).
+* Alternative chain is similarly a special case of object enhancement, as the signature is identical and the dispatch workflow is nearly identical, albeit with an extra check included.
```

Comment:
The casing of "Object enhancement" was simply a by-product of it being the first word of a bulleted point; it's not a specific vocabulary definition we have provided. As such, I'm recommending here we remove the special casing.